### DISCUSSION TITLE
Use custom model functions as column and make it searchable and sortable

### DISCUSSION THREAD

**Discussion by Restingo** [rappasoft/laravel-livewire-tables#320] [Q&A] [answered]
I've got a function like `getDefaultEmail()` that returns an email formatted as clickable link. I want to include this into my table and also make it searchable and sortable. Currently it is loaded into my model as a trait. How can I do that?

**Comment by rappasoft** [comment, marked as answer]
```php
Column::make('E-mail', 'email')
    ->sortable()
    ->searchable()
    ->format(function($value, $column, $row) {
        return $row->getDefaultEmail();
    })
    ->asHtml(),
```

https://github.com/rappasoft/laravel-livewire-tables/wiki/Using-the-built-in-cell-formatting
